### ISSUE TITLE
Unit Testing: Homepage callout image doesn’t resize on an iPad or iPhone display

### ISSUE THREAD

**Issue by simonjvardy** [simonjvardy/Aviation-Consultancy#60]
**Describe the bug**

Homepage callout image doesn’t resize on an iPad or iPhone display

**To Reproduce**
1. Go to https://simonjvardy.github.io/Aviation-Consultancy/
2. Click on 'Home'
3. See error

**Expected behavior**
The background image is expected to be responsive and resize automatically to an appropriate size based on the image dimensions.

**Screenshots**
![image](https://user-images.githubusercontent.com/65361514/91184534-a3027280-e6e4-11ea-9b04-200157fe5d5e.png)

![image](https://user-images.githubusercontent.com/65361514/91184597-b7466f80-e6e4-11ea-9c07-73680b5162f5.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
 - Device: iPad Pro 12.9 Gen1
 - OS: iPadOS 13.6.1
 - Browser Chrome, Firefox and Safari

 - Device: iPhone XS Max
 - OS: iOS 13.6.1
 - Browser Chrome, Firefox and Safari

**Additional context**
Viewing the website using Chrome Developer tools on a desktop browser shows the site as expected when viewing  different screen sizes
![image](https://user-images.githubusercontent.com/65361514/91184994-2d4ad680-e6e5-11ea-838b-56737962a03d.png)

![image](https://user-images.githubusercontent.com/65361514/91185155-5e2b0b80-e6e5-11ea-9d6f-606fb5d542c9.png)

**Comment by simonjvardy** [comment]
This doesn't work on the iPad, only th iPhone. Further investigation required

**Comment by simonjvardy** [comment]
Added the following CSS code to try to fix this problem:

```
* -------------------------------------Responsive Design - Apple Products Fix */
/* Apple devices do not support background-attachment: fixed; which has caused the background images to display incorrectly*/
/* These media queries are a workaround to try to fix issue #60 by adding smaller image versions and setting the */
/* background-attachment: scroll; for all devices smaller than 1120px device width which covers most iPhones and iPads */
/* in portrait and landscape modes */


@media screen and (max-width: 1120px) {
    .callout-container {
        background: url('../images/aircraft-hangar-960w.jpg') no-repeat center scroll;
        background-size: cover;
    }
    .testimonial-carousel {
        background: url('../images/above-the-clouds.jpg') no-repeat center scroll;
        background-size: cover;
    }
    .our-services-hero-container {
        background: url('../images/aircraft-engine-960w.jpg') no-repeat center scroll;
        background-size: cover;
    }
    .about-us-hero-container {
        background: url('../images/aircraft-hangar-front-web-960w.jpg') no-repeat center scroll;
        background-size: cover;
    }
    .contact-us-hero-container {
        background: url('../images/aircraft-taxi-night-960w.jpg') no-repeat center scroll;
        background-size: cover;
    }
}
```

**Comment by simonjvardy** [comment]
Issue fixed as far as it can be for this project. The smaller images work better on mobile devices and the iPhone / iPad displays the images as expected now
![image](https://user-images.githubusercontent.com/65361514/91912374-ea6f9c80-ecaa-11ea-80c1-1ae7c2117c12.png)

![image](https://user-images.githubusercontent.com/65361514/91912396-f0fe1400-ecaa-11ea-92d2-0ca8dcc0a9ae.png)
